pkgs/development/ocaml-modules/json-data-encoding/bson.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildDunePackage, json-data-encoding, ocplib-endian, crowbar }:
+
+buildDunePackage {
+  pname = "json-data-encoding-bson";
+
+  inherit (json-data-encoding) version src useDune2 doCheck;
+
+  propagatedBuildInputs = [
+    json-data-encoding
+    ocplib-endian
+  ];
+
+  checkInputs = [
+    crowbar
+  ];
+
+  meta = {
+    homepage = "https://gitlab.com/nomadic-labs/json-data-encoding";
+    description = "Type-safe encoding to and decoding from JSON (bson support)";
+    license = lib.licenses.lgpl3;
+    maintainers = [ lib.maintainers.ulrikstrid ];
+  };
```

Comment:
You may reuse parts of `json-data-encoding.meta` instead of copying them.

Reply:
Do you intend to remove the duplicated parts or shall I merge as is?

Reply:
I won't have time before Monday, but it's up to you. If you want me to reuse it I'll do that

Reply:
Fixed